test_conformance/computeinfo/main.cpp

```diff
@@ -480,8 +599,9 @@ int getConfigInfo(cl_device_id device, config_info* info)
     return err || size_err;
 }
 
-void dumpConfigInfo(cl_device_id device, config_info* info)
+void dumpConfigInfo(config_info* info)
 {
+    int number_of_version_items = 0;
```

Comment:
Looks like this is only used in one case of the switch, could declare it there on line 899?